Java/CoreCrypto/src/main/java/com/breadwallet/corecrypto/Wallet.java

```diff
@@ -167,7 +167,7 @@ public int hashCode() {
 
     /* package */
     void setDefaultFeeBasis(com.breadwallet.crypto.TransferFeeBasis feeBasis) {
-        defaultFeeBasis = TransferFeeBasis.from(feeBasis);
+        core.setDefaultFeeBasis(TransferFeeBasis.from(feeBasis).getCoreBRFeeBasis());
```

Comment:
A Transfer's feeBasis is settable after the fact?  I don't think that is the case and probably in the getter+setter it should not be called 'Default'.  Is this a consequence of Transfer::estimateFee?